group1/communication/node/main.js

```diff
@@ -173,6 +318,25 @@ var messages = {
                 });
         }
 };
+var indvroom={
+    'save':function (creator_id, second_user) {
+           indvroom_template={
+               data:{
```

Comment:
Shouldn't "data" be surrounded with single quotes?  There should be a space between ":" and "{"